std/json.d

```diff
@@ -703,7 +703,7 @@ JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOpti
 if (isInputRange!T)
 {
     import std.ascii : isWhite, isDigit, isHexDigit, toUpper, toLower;
-    import std.utf : toUTF8;
+    import std.utf : encode, Yes;
```

Comment:
FWIW any module using template flags is supposed to export `Yes` as well, so this is really personal preference or personal purism ;-)